samples/bot-conversation-sso-quickstart/csharp_dotnetcore/M365Agent/m365agents.local.yml

```diff
@@ -27,15 +28,13 @@ provision:
       name: bot-conversation-sso-quickstart${{APP_NAME_SUFFIX}}
     # Write the information of created resources into environment file for
     # the specified environment variable(s).
-    writeToEnvironmentFile: 
+    writeToEnvironmentFile:
       teamsAppId: TEAMS_APP_ID
 
   - uses: script
     with:
-      run:
-        # echo "::set-teamsfx-env MICROSOFT_APP_TYPE=SingleTenant";
-        echo "::set-teamsfx-env MICROSOFT_APP_TYPE=MultiTenant";
-        echo "::set-teamsfx-env MICROSOFT_APP_TENANT_ID=${{AAD_APP_TENANT_ID}}";
+      run: echo "::set-teamsfx-env MICROSOFT_APP_TYPE=SingleTenant"; echo
+        "::set-teamsfx-env MICROSOFT_APP_TENANT_ID=${{AAD_APP_TENANT_ID}}";
```

Comment:
Please correct the format like below to maintain consistency

<img width="376" height="42" alt="Image" src="https://github.com/user-attachments/assets/ac58918a-e0d8-4395-89c6-5845ce10749d" />